1-exercises/A-accessing-values/exercise2.js

```diff
@@ -17,7 +17,7 @@ let capitalCities = {
 */
 
 let myCountry = "UnitedKingdom";
-let myCapitalCity; // complete the code
+let myCapitalCity = capitalCities["UnitedKingdom"] // complete the code
```

Comment:
here you can use the variable `myCountry` to fetch the capital for the country, as the variable is assigned the string `UnitedKingdom` :)